### PR TITLE
using put_file method instead of python shutil.copyfileobj

### PR DESCRIPTION
**Pull request by racpatel**
Signed-off-by: Rachana Patel <racpatel@redhat.com>

### CODE REVIEW

**Comment by vasukulkarni** [comment]
LGTM, I think you got the dest path correct.

**Comment by vasukulkarni** [comment]
@zmc @djgalloway FYI

File: teuthology/task/kernel.py [hunk 1]
```diff
@@ -342,17 +342,7 @@ def download_kernel(ctx, config):
...
-                args=[
-                    'python', '-c',
-                    'import shutil, sys; shutil.copyfileobj(sys.stdin, file(sys.argv[1], "wb"))',
-                    remote_pkg_path(role_remote),
-                    ],
-                wait=False,
-                stdin=f
-                )
-            procs[role_remote.name] = proc
+            role_remote.put_file(src,remote_pkg_path(role_remote))
```
Comment (smithfarm) [inline teuthology/task/kernel.py]: nit: insert a space after the comma in `src,`

Comment (kshtsk) [reply teuthology/task/kernel.py]: this PR is to rh downstream, the analogue upstream PR is already merged

**Comment by kshtsk** [comment]
I guess you can close this PR and cherry-pick it from upstream or rebase 'rh' to master?

**Pull request closed without merging**